analytic_engine/src/sst/parquet/async_reader.rs

```diff
@@ -105,29 +142,54 @@ impl<'a> Reader<'a> {
             &meta_data.custom().bloom_filter,
         )?;
 
-        debug!(
-            "fetch_record_batch row_groups total:{}, after filter:{}",
+        info!(
+            "fetch_record_batch row_groups total:{}, after filter:{}, row_group_num_per_reader:{}",
             meta_data.parquet().num_row_groups(),
-            filtered_row_groups.len()
+            filtered_row_groups.len(),
+            read_parallelism,
         );
 
+        // TODO: now `batch_size` is equal to `num_rows_per_row_group`, other situations
```

Comment:
I  can't understand this TODO, what issue are you trying to solve?

Reply:
Has written an new TODO.